### ISSUE TITLE
The construction `נמנע מ + Infinitive` 

### ISSUE THREAD

**Issue by Hilla-Merhav** [IAHLT/UD_Hebrew#25]
@amir-zeldes

I wonder how we should analyze the construction `נמנע מ + Infinitive` (for example: נמנע מלהכריע). It seems to me like a classic `cobl`, but we can't tag it this way – if I understood correctly, when we see a `cobl`, we have nothing to do but to tag it as `advcl`, is that right?

If we do tag it as `advcl`, I assume **mi-** (מ-) here should be an SCONJ governed by `mark`?

Unless we don't look at it as a clause? On HTB I found in most cases the infinitive tagged as an `obl`, but I guess it's wrong. If the example is נמנע מלהכריע:

**7 occurrences on HTB**
`obl`(nimna, lehaxria) 
`case`(lehaxria, mi)

But the one case it's `advcl`, the infinitive governs a `case` rather than a `mark`.

**1 occurrence on HTB**
`advcl`(nimna, lehaxria) 
`case`(lehaxria, mi)

What do you recommend doing?

**Comment by amir-zeldes** [comment]
Yep, you understood it exactly - it's meant to be advcl + mark. The HTB cases should be fixed so I'll leave this issue as a reminder for that, but it's a more general problem than nimna (all VERBs governing an infinitive as obl will have to be made into advcl+mark). The only alternative is to use a subtype (some people do things like `advcl:arg`, which also looks pretty terrible to me) but as a major relation it has to be `advcl` sadly.

@dan-zeman - I thought I'd draw your attention to this again, it's something I mentioned at Dagstuhl and last meeting we had as well: oblique argument clauses being tagged as `advcl` is one of those eyesores that I always get complaints about whenever I teach UD in a Linguistics forum or talk with linguists who look at UD in depth...

**Comment by dan-zeman** [comment]
> I thought I'd draw your attention to this again, it's something I mentioned at Dagstuhl and last meeting we had as well: oblique argument clauses being tagged as `advcl` is one of those eyesores that I always get complaints about whenever I teach UD in a Linguistics forum or talk with linguists who look at UD in depth...

I agree that the relation names (and sometimes also rules) for clausal dependents are not optimal. We can talk about this at the UD guidelines meeting but I'm afraid that changing this would mean re-designing the way how the UD relation taxonomy is organized, which is not too likely to happen...

**Comment by amir-zeldes** [comment]
Thanks for the quick reply Dan - I'm definitely not suggesting overturning this guideline anytime soon, and I don't have a concrete suggestion on what to do/how yet, but I wanted to keep this issue alive and provide some evidence that other people find this problematic as well.
